### PR TITLE
[UTOPIA-773]add additional risk tab

### PR DESCRIPTION
**Pull request by binggaofw**
# Description

This PR includes the following proposed change(s):

- add additional risk tab, make a list component so it can reuse by different tab

Screen Shot 2023-02-14 at 7 29 17 PM

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Development Dependency Working Agreement
- [x] My code DOES NOT include the importing of new dependencies into the DPIA ecosystem
- [ ] My code DOES include the importing of new dependencies into the DPIA ecosystem
**If new dependencies are being introduced to the DPIA ecosystem:**
- [ ] The functionality of the dependency drastically reduces code complexity and makes my changes more easily maintainable and readible 
- [ ] The dependency being introduced does not contain multiple layers of nested dependencies introducing maintainability complexity to the DPIA ecosystem

## Frontend Development Changes
- [ ] N/A
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to project documentation or diagrams that reflect my changes
- [ ] New and existing unit tests pass locally with my changes
- [ ] My code follows Airbnb React Style Guidelines

## API Development Changes
- [x] N/A
- [ ] I have performed a self-review of my own code
- [ ] My code follows standards and practices outlined in the BC Government API Development Guidelines
- [ ] New and existing unit tests pass locally with my changes
- [ ] My changes includes Swagger documentation updates that reflect the changes I am introducing

## Definition of Done

Definition of Done

### CODE REVIEW

File: src/frontend/src/routes/router.tsx [hunk 1]
```diff
@@ -44,6 +45,7 @@ const Router = () => {
         <Route path="/pia/intake/:id/:title" element={<PIADetailPage />} />
       </Route>
       <Route path="/not-authorized" element={<Unauthorized />} />
+      <Route path="/tab2" element={<PIAAdditionalRisks />} />
```
Comment (binggaofw) [inline src/frontend/src/routes/router.tsx:48]: same as last pr, this route just for test purpose, I will delete it before merge the pr to main

File: src/frontend/src/components/public/PIAAdditionalRisks/index.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
+import { useState } from 'react';
+import Messages from './messages';
+import InputText from '../../common/InputText/InputText';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faPlus, faTrash } from '@fortawesome/free-solid-svg-icons';
+import { useNavigate } from 'react-router-dom';
+import List from '../../common/List';
+
+export interface TableColProps {
```
Comment (Bottle7) [inline src/frontend/src/components/public/PIAAdditionalRisks/index.tsx:9]: This interface is identical to the above InputTextProps and I believe that it is used for the same data. I would recommend using this interface in the List **OR** using the List's InputTextProps interface.

File: src/frontend/src/components/common/List/index.tsx [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+              <tbody>
+                {data.map((items, idx) => (
+                  <tr key={idx}>
+                    {items.map((item, index) => (
+                      <td key={index}>
+                        <div className=" row">
+                          <div>
+                            {index === 0 ? (
+                              <label className="mt-3 ">
+                                {'Risk' + ' ' + (idx + 1)}
```
Comment (Bottle7) [inline src/frontend/src/components/common/List/index.tsx:57]: Use of this hard-coded string 'Risk' makes this much less reusable.

Comment (binggaofw) [reply src/frontend/src/components/common/List/index.tsx:57]: yes, i will change this. will pass label as props to list

**Review by kushal-arora-fw — COMMENTED** [review]
Do not see form state for this Tab as well. Please use the model from the backend as discussed for consistency.

**Comment by binggaofw** [comment]
close it now and reopen later

**Pull request closed without merging**